### PR TITLE
Remove unused `delegation_pool` module from supra_framework

### PR DESCRIPTION
**Pull request by nizam-supraoracles**
- Removed `delegation_pool` module from supra_framework.
- Also cleanup this function's uses in another module.
- Auto generated file updated based on this changes

### CODE REVIEW

**Review by sjoshisupra — APPROVED** [review]

File: crates/aptos-rosetta/src/client.rs [hunk 1]
```diff
@@ -528,13 +528,7 @@ impl RosettaClient {
...
         keys.insert(delegator, private_key);
 
-        let operations = vec![Operation::add_delegated_stake(
-            0,
-            None,
-            delegator,
-            AccountIdentifier::base_account(pool_address),
-            amount,
-        )];
+        let operations = vec![];
```
Comment (sjoshisupra) [inline crates/aptos-rosetta/src/client.rs]: When this vector is empty, no operation is performed. Doesn't this mean that the test is practically removed?

File: crates/aptos-rosetta/src/client.rs [hunk 2]
```diff
@@ -567,13 +561,7 @@ impl RosettaClient {
...
         keys.insert(delegator, private_key);
 
-        let operations = vec![Operation::unlock_delegated_stake(
-            0,
-            None,
-            delegator,
-            AccountIdentifier::base_account(pool_address),
-            amount,
-        )];
+        let operations = vec![];
```
Comment (sjoshisupra) [inline crates/aptos-rosetta/src/client.rs]: Same as above.

File: crates/aptos-rosetta/src/construction.rs [hunk 3]
```diff
@@ -940,13 +940,7 @@ pub fn parse_delegation_pool_add_stake_operation(
...
     let amount: u64 = parse_function_arg("add_delegated_stake", args, 1)?;
 
-    Ok(vec![Operation::add_delegated_stake(
-        0,
-        None,
-        delegator,
-        AccountIdentifier::base_account(pool_address),
-        Some(amount),
-    )])
+    Ok(vec![])
```
Comment (sjoshisupra) [inline crates/aptos-rosetta/src/construction.rs]: Same as above.

File: crates/aptos-rosetta/src/construction.rs [hunk 4]
```diff
@@ -988,13 +976,7 @@ pub fn parse_delegation_pool_withdraw_operation(
...
     let amount: u64 = parse_function_arg("withdraw_undelegated", args, 1)?;
 
-    Ok(vec![Operation::withdraw_undelegated_stake(
-        0,
-        None,
-        delegator,
-        AccountIdentifier::base_account(pool_address),
-        Some(amount),
-    )])
+    Ok(vec![])
```
Comment (sjoshisupra) [inline crates/aptos-rosetta/src/construction.rs]: same as above

File: crates/aptos-rosetta/src/construction.rs [hunk 5]
```diff
@@ -964,13 +958,7 @@ pub fn parse_delegation_pool_unlock_operation(
...
     let amount: u64 = parse_function_arg("unlock_delegated_stake", args, 1)?;
 
-    Ok(vec![Operation::unlock_delegated_stake(
-        0,
-        None,
-        delegator,
-        AccountIdentifier::base_account(pool_address),
-        Some(amount),
-    )])
+    Ok(vec![])
```
Comment (sjoshisupra) [inline crates/aptos-rosetta/src/construction.rs]: same as above

**Review by sjoshisupra — CHANGES_REQUESTED** [review]

File: crates/aptos-rosetta/src/client.rs (hunk 1 above)
Comment (nizam-supraoracles) [reply crates/aptos-rosetta/src/client.rs]: Okay, Actually we need to remove entire test for **delegation pool** module from `aptos-rosetta`. 
I made the changes

**Review by sjoshisupra — APPROVED** [review]

**Pull request merged**